vagrant/provisioning/arkcase-dev-facts.yml

```diff
@@ -0,0 +1,321 @@
+#################################
+## host-specific settings
+#################################
```

Comment:
keep everything just like it is in the old Vagrant VM, except for what must be changed due to using MariaDB (instead of MySQL) and Samba (instead of ApacheDS)